ocr_service/Dockerfile

```
@@ -0,0 +1,52 @@
+COPY requirements.txt .
+# Download all dependencies including implicit ones
+RUN pip wheel --no-cache-dir --wheel-dir /app/wheels -r requirements.txt
+
+# Final stage
+FROM python:3.9-slim
+
+WORKDIR /app
+
+# Install only runtime dependencies in a single layer
+# Combine the Tesseract installation with English language support
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)